koku/masu/providers/status.py

```diff
@@ -30,6 +31,18 @@
 class ProviderStatus(ProviderStatusAccessor):
     """Provider Status."""
 
+    def __repr__(self):
```

Comment:
Adding `__repr__()` and `__str__()` methods isn't strictly necessary. This is mainly just a quality-of-life improvement I've tacked on because it was useful in developing this bugfix.